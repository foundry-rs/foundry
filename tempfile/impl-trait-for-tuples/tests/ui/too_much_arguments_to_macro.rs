#[impl_trait_for_tuples::impl_for_tuples(1, 2, 3)]
trait Test {}

fn main() {}
