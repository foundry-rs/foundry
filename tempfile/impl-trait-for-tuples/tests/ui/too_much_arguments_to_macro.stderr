error: Too many arguments given to the macro!
 --> tests/ui/too_much_arguments_to_macro.rs:1:1
  |
1 | #[impl_trait_for_tuples::impl_for_tuples(1, 2, 3)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `impl_trait_for_tuples::impl_for_tuples` (in Nightly builds, run with -Z macro-backtrace for more info)
